src/babashka/neil.clj

```diff
@@ -364,6 +362,11 @@
    :target-dir
    :overwrite
 
+   ; template deps overrides
+   :local/root
+   :git/url
+   :git/sha
```

Comment:
From the [deps-new docs](https://github.com/seancorfield/deps-new#more-general-usage):
> `:local/root` and `git`-based coordinates are supported, but Maven/Clojars coordinates are not.